lib/src/screens/send/widgets/send_card.dart

```diff
@@ -331,6 +331,7 @@ class SendCardState extends State<SendCard>
                             ],
                           ),
                         )),
+                    if (sendViewModel.balanceViewModel.disableFiat)
```

Comment:
shouldn't this be negated? i.e (`if (!sendViewModel.balanceViewModel.disableFiat)`)